libnavui-base/src/main/java/com/mapbox/navigation/ui/base/model/TurnIconResources.kt

```diff
@@ -0,0 +1,267 @@
+package com.mapbox.navigation.ui.base.model
+
+/**
+ * An interface describing all maneuver icons.
+ */
+interface TurnIconResources {
```

Comment:
Can we name it as `BaseTurnIconResources` rather than using it as an alias in import statements?

Reply:
What's wrong with `TurnIconResources` and using import aliases?

Reply:
There is nothing wrong with it, but again as pointed above, I think directly using the name in the original file improves the readability because import statements are usually collapsed and we don't tend to look at it. If someone were to find this file they won't find it because the alias is pretty much hidden in the collapse and this file actually exists by a different name. This happened with me in this PR.